client/src/components/simple-components/ButtonsLinks.tsx

```diff
@@ -59,6 +59,14 @@ const SmallRoundedButton = styled.button<{ color?: ColorType }>`
     ${(props) => ColorCss(props.color)}
 `;
 
+const LargeHalfRoundedButton = styled.button<{ color?: ColorType }>`
+  ${ButtonCss}
+  ${HalfRoundedCss}
+${LargeCss}
+
+${(props) => ColorCss(props.color)}
```

Comment:
Looks like you need to run prettier

Reply:
lol i have no idea why it looks like that it didn't change after i ran prettier, but i fixed it